api/v1alpha1/openapi.yaml

```diff
@@ -1013,6 +1013,50 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+
+  /api/v1/devices/summary:
```

Comment:
This overloads `/api/v1/devices/{name}`. ATM, we only allow device fingerprints for names and those do not conflict, of course. However, we do have the request to support more human-readable names (not aliases) and may solve that through user-defined generator functions (e.g. based on hostname). In that case there is a residual risk of conflict (e.g. a host being named "summary").

Reply:
Yes agree, it makes issues with the CLI as well